.mergify.yml

```
@@ -4,23 +4,35 @@ pull_request_rules:
       - author~=dependabot\[bot\]|dependabot-preview\[bot\]
       - status-success=Travis CI - Pull Request
       - status-success=continuous-integration.azure-devops
+      - or:
+          - -title~=(?i)pyinstaller
+          - and:
+            - status-success=codebuild/custom/pr
+            - status-success=codebuild/default/pr
+            - title~=(?i)pyinstaller
     actions:
       review:
         type: APPROVE
-  # For salt-formula updates, trigger codebuild, merge on codebuild status
+
+  # For salt-formula or pyinstaller updates, trigger codebuild, merge on codebuild status
   - name: write issue comment to start codebuild job
     conditions:
       - author~=dependabot\[bot\]|dependabot-preview\[bot\]
-      - label=submodules
-      - "#approved-reviews-by>=1"
+      - or:
```

i think we can flip this, so both cases require the approval rule first:

```suggestion
      - and:
        - "#approved-reviews-by>=1"
        - or:
          - label=submodules
          - title~=(?i)pyinstaller
```